changelogs/fragments/587-ec2_instance-default-instance-type-launch-template.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - ec2_instance - Add a condition to handle default ```instance_type``` value
+    for fix breaking on instance creation with launch template (https://github.com/ansible-collections/amazon.aws/pull/587).
```

Comment:
Could you please note the deprecation of the default value in the changelog as well?  Otherwise this should be good once the sanity ignore for 2.9 is added - thanks!